client/stylesheets/_toCat.scss

```diff
@@ -5445,4 +5445,33 @@ On click the images will be displayed at normal size to complete the effect
 
 .listingPayCash i {
 	margin-right: 5px;
-}
\ No newline at end of file
+}
+
+
+/** Amount Profile **/
+
+.profileUserAmount ul {
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces